x-pack/plugins/endpoint/server/services/endpoint/alert_query_builders.ts

```diff
@@ -7,9 +7,9 @@ import { KibanaRequest } from 'kibana/server';
 import { EndpointAppConstants } from '../../../common/types';
 import { EndpointAppContext, AlertRequestParams, JSONish } from '../../types';
 
-export const buildAlertListESQuery = async (
+export const buildAlertListESQuery: (
```

Comment:
This fn used no `await` so I removed `async`. Might look into adding lint to enforce this.